Raven.Tests/AllExceptionsAreSerializaable.cs

```diff
@@ -25,6 +25,9 @@ public void AllExceptionsAreGood()
 
                 foreach (var customException in customExceptions)
                 {
+                    if (customException.Name == "OptimisticConcurrencyViolationException")
```

Comment:
I think it would be safer just to make this exception serializable, also in situations like this use nameof(T) or typeof(T).Name